rest-server/README.md

```diff
@@ -175,30 +236,60 @@ Configure the rest server port in [services-configuration.yaml](../cluster-confi
 
     *Response if succeeded*
     ```
-	Status: 200
-	
+	  Status: 200
+
     {
       "message": "remove successfully"
     }
     ```
 
-    *Response if an error occured*
+	  *Response if not authorized*
     ```
-    Status: 500
+    Status: 401
 
     {
-      "error": "RemoveFailed",
-      "message": "remove failed"
+      "code": "UnauthorizedUserError",
+      "message": "Guest is not allowed to do this operation."
     }
     ```
-	
-	*Response if not authorized*
+
+	  *Response if user has no permission*
+    ```
+    Status: 403
+
+    {
+      "code": "ForbiddenUserError",
+      "message": "Non-admin is not allow to do this operation."
+    }
     ```
-    Status: 401
+
+	  *Response if an admin will be removed*
+    ```
+    Status: 403
 
     {
-      "error": "NotAuthorized",
-      "message": "not authorized"
+      "code": "RemoveAdminError",
+      "message": "Admin $username is not allowed to remove."
+    }
+    ```
+
+    *Response if updated user does not exist*
+    ```
+    Status: 404
+
+    {
+      "code": "NoUserError",
+      "message": "User $username not found."
+    }
+    ```
+
+    *Response if a server error occured*
+    ```
+    Status: 500
+
+    {
+      "code": "UnknownError",
+      "message": "*Upstream error messages*"
```

Comment:
`*` doesn't work as italics in a code block.

Reply:
@abuccts I just want to express about this message is different from any other hard-coded error message. How about `"$upstreamErrorMessages"` or any good idea?

Reply:
The original is OK if you doesn't mean to change the font style.